src/app/activity/bounty/Create.ts

```diff
@@ -91,104 +96,49 @@ export const createBounty = async (createRequest: CreateRequest): Promise<any> =
             criteria,
             dueAt,
             guildMember,
-            null);
+            null,
+            createRequest.createdInChannel);
     } else {
-        const owedTo = await DiscordUtils.getGuildMemberFromUserId(createRequest.owedTo, createRequest.guildId);
+        owedTo = await DiscordUtils.getGuildMemberFromUserId(createRequest.owedTo, createRequest.guildId);
         newBounty = await createDbHandler(
             createRequest,
             null,
-            null,
-            null,
+            'IOU for work already done',
```

Comment:
This should probably be null? Keeping track of this over time may get confusing, and i'm thinking we can always overwrite it when constructing the canonical card

If you feel strongly about including it in the initial bounty data, let's put this value behind a static constant